internal/state/execution.go

```diff
@@ -195,7 +195,19 @@ func (blockExec *BlockExecutor) ProcessProposal(
 // Validation does not mutate state, but does require historical information from the stateDB,
 // ie. to verify evidence from a validator at an old height.
 func (blockExec *BlockExecutor) ValidateBlock(state State, block *types.Block) error {
-	err := validateBlock(state, block)
+	err := validateBlock(state, block, false)
```

Comment:
Hmm. why is this popping here?

Reply:
Please ignore this comment: I realize it's a change added by this PR, as we're not checking BFT Time from blocksync